lib/wibox/widget/graph.lua

```diff
@@ -975,6 +982,9 @@ for _, prop in ipairs(properties) do
     end
     if not graph["get_" .. prop] then
         graph["get_" .. prop] = function(_graph)
+            if properties_defaults[prop] ~= nil and  _graph._private[prop] == nil then
```

Comment:
Instead of bothering with this automagic, that violates the `set_prop(v); get_prop() == v` property, I'd just set the properties once in `graph:new()` and be done with it.

Reply:
Yes, this would work

Reply:
Okay, I will do that then.